src/config/Config.zig

```diff
@@ -400,9 +400,17 @@ theme: ?Theme = null,
 /// Background color for the window.
 background: Color = .{ .r = 0x28, .g = 0x2C, .b = 0x34 },
 
+/// Background color for the titlebar. This is only takes effect if
+/// window-theme is set to ghostty.
+@"titlebar-background": ?Color = null,
```

Comment:
I think to fit in with the rest of our configs this should be `window-titlebar-background` (and likewise for foreground). And they should be down in the `window-`-prefixed section.